command/command.go

```diff
@@ -23,9 +23,15 @@ var version = "v1.0.0-rc.1"
 // empty context
 var nocontext = context.Background()
 
+func new() {
```

Comment:
Use context.WithCancel or context.WithTimeout instead of context.Background() to provide more control over the execution context.